libs/blocks/icon-block/icon-block.css

```diff
@@ -53,13 +61,39 @@
   width: auto;
 }
 
+.icon-block.bio .foreground .icon-area {
+  width: 80px;
+  height: 80px;
+  position: relative;
+  overflow: hidden;
+  border-radius: 50%;
+}
+
+.icon-block.bio .foreground .icon-area img {
+  object-fit: cover;
+}
+
+.five-up .icon-block:not(.full-width) .foreground .icon-area {
+  height: 44px;
+}
+
+.five-up .icon-block:not(.full-width) .foreground .icon-area img {
+  height: 44px;
+}
+
+.section.five-up:has(.icon-block) {
+  grid-template-columns: repeat(auto-fit, minmax(142px, 1fr));
+}
```

Comment:
Changing the grid on a the block level doesn't seem like the best solution. It's jarring for authors who don't know why this grid looks different than the other five-up grid. Perhaps instead add an option for a smaller container to the section metadata

Reply:
okay thanks for the feedback, I moved to section metadata